x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/selectors.ts

```diff
@@ -359,9 +360,34 @@ export const getIsolationRequestError: (
   }
 });
 
+export const getActivityLogDataPaging = (
+  state: Immutable<EndpointState>
+): Immutable<Omit<EndpointState['endpointDetails']['activityLog'], 'logData'>> => {
+  return {
+    page: state.endpointDetails.activityLog.page,
+    pageSize: state.endpointDetails.activityLog.pageSize,
+  };
+};
+
+export const getPreviousLogData = (state: Immutable<EndpointState>): Immutable<ActivityLog> => {
```

Comment:
Is getLastLoadedResourceState useful for you in this scenario?

Reply:
I guess so. Will refactor in the UX PR.

Reply:
done 955bc2c2339d4b4f89b4ff6336d5569d507fe20b